packages/server/api/src/app/database/migrations/1757331587268-AddContentTypeToFolder.ts

```diff
@@ -0,0 +1,39 @@
+import { logger } from '@openops/server-shared';
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddContentTypeToFolder1757331587268 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    logger.info('AddContentTypeToFolder1757331587268: starting');
+
+    await queryRunner.query(`
+      ALTER TABLE "folder"
+      ADD COLUMN IF NOT EXISTS "contentType" varchar;
+    `);
+
+    await queryRunner.query(`
+      UPDATE "folder" 
+      SET "contentType" = 'WORKFLOW'
+      WHERE "contentType" IS NULL;
+    `);
+
+    await queryRunner.query(`
+      ALTER TABLE "folder"
+      ALTER COLUMN "contentType" SET NOT NULL;
+    `);
+
+    await queryRunner.query(`
```

Comment:
adding a new index, you should be able to have the same folder name for different contenttypes